packages/lmi/README.md

```diff
@@ -43,14 +43,14 @@ A simple example of how to use the library with default settings is shown below.
 
 ```python
 from lmi import LiteLLMModel
-from aviary import Message
+from aviary.core import Message
 
 llm = LiteLLMModel()
 
 messages = [Message(content="What is the meaning of life?")]
 
 completion = await llm.call_single(messages)
-assert completion.text == "42"
+# assert completion.text == "42"
```

Comment:
Let's name the output of `call_single` as `result`, since its type is `LLMResult`

Also, why comment this line?